extension/llm/src/function/create_embedding.cpp

```diff
@@ -94,20 +94,20 @@ static uint64_t parseDimensions(std::shared_ptr<Expression> dimensionsExpr,
 
 static std::unique_ptr<FunctionBindData> bindFunc(const ScalarBindFuncInput& input) {
     std::optional<uint64_t> dimensions = std::nullopt;
-    std::optional<std::string> region = std::nullopt;
+    std::optional<std::string> regionOrEndpoint = std::nullopt;
```

Comment:
How about we call this stringConfig/numConfig here?
provider.configure can then use the generic names instead.

Reply:
For providers, instead of giving an error saying `dimensions not expected`, use the same style as the other PR.
```
Unexpected arguments:
actual:      (string, string, string, bool)
expected: (string, string, string, string)
                   (string, string, string, int, string)
```